Corona-Warn-App/src/main/res/values-bg/strings.xml

```diff
@@ -999,9 +999,9 @@
     <string name="submission_qr_code_scan_invalid_dialog_button_negative">"Отказ"</string>
 
     <!-- QR Code Scan Screen -->
-    <string name="submission_qr_code_scan_title">"Позиционирайте QR кода в рамката."</string>
+    <string name="qr_code_scan_title">"Позиционирайте QR кода в рамката."</string>
```

Comment:
Please only change it in de and the defaults, otherwise it will cause merge conflicts.